src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -19,11 +18,11 @@ import { useUpdateActiveEnterpriseForUser, useEnterpriseCustomerConfig } from '.
 import { pushUserCustomerAttributes } from '../../utils/optimizely';
 
 const EnterprisePage = ({ children, useEnterpriseConfigCache }) => {
+  const { authenticatedUser: user } = useContext(AppContext);
```

Comment:
[context, pun intended] By subscribing to `AppContext` here to pull the `authenticatedUser` versus using `getAuthenticatedUser` directly.